File: C/Fuck/fuction/swap.c
#include <stdio.h>

void swap(int* a, int* b)
{
    int c = *a;
    *a = *b;
    *b = c;
}

int main()
{
    int a = 1;
    int b = 2;
    printf("before swap: a = %d, b = %d\n", a, b);
    swap(&a, &b);
    printf("after swap: a = %d, b = %d", a, b);

    return 0;
}

File: Java/assignments/s1_java_basics/10.20 in class/src/swap.java
public class swap {
  public static void main(String[] args) {
    int a = 1;
    int b = 2;
    System.out.println("a: " + a);
    System.out.println("b: " + b);
    a = a + b;
    b = a - b;
    a = a - b;
    System.out.println("a: " + a);
    System.out.println("b: " + b);
  }
}
